R/pkg/R/functions.R

```diff
@@ -259,6 +259,20 @@ setMethod("column",
           function(x) {
             col(x)
           })
+#' corr
+#'
+#' Computes the Pearson Correlation Coefficient for two Columns.
+#'
+#' @rdname corr
+#' @name corr
+#' @family math_funcs
+#' @export
+#' @examples \dontrun{corr(df$c, df$d)}
+setMethod("corr", signature(x = "Column", col1 = "Column", col2 = "missing", method = "missing"),
```

Comment:
this signature looks confusing. Maybe change the generic function definition of "corr" is better:
```
setGeneric("corr", function(x, ...) {standardGeneric("corr") })

setMethod("corr",
          signature(x = "DataFrame"),
          function(x, col1, col2, method = "pearson") {
          ...
}

setMethod("corr", signature(x = "characterOrColumn"),
                  function(x, col2) {
                  ...
}
```